frontend/src/components/pages/Appointments.js

```diff
@@ -69,19 +140,46 @@ function Appointments() {
     );
   };
 
-  const UpcomingTab = () => {
-    // Example, feel free to remove or change
-    return null;
+  const Appointments = (props) => {
```

Comment:
instead of just passing in props you can use [object destructuring](https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0) (cont)
```suggestion
  const Appointments = ({data}) => {
```